docs/pwa.md

```
@@ -9,40 +14,49 @@
+const CACHE_NAME = `citronics-${CACHE_VERSION}`
 ```
 
-> When you make breaking changes to cached assets, increment the version: `eventhub-v2`.  
-> The activate handler automatically cleans up old caches.
+> When you make breaking changes to cached assets, bump the version: `v2`, `v3`, …  
+> The `activate` handler automatically deletes **all** caches whose name ≠ `CACHE_NAME`,  
+> including the legacy `eventhub-*` caches from the pre-rebrand era (migration is automatic).
 
 ### Caching strategy — Network First
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)